operator/v1/types_ingress.go

```diff
@@ -99,6 +99,13 @@ type IngressControllerSpec struct {
 	// the generated certificate's CA will be automatically integrated with the
 	// cluster's trust store.
 	//
+	// If a wildcard certificate is used and shared by multiple HTTP/2
+	// enabled routes (which implies APLN) then clients (i.e., notably
```

Comment:
"APLN" → "ALPN".

Reply:
Done in https://github.com/openshift/api/pull/654/commits/f48795ed59406cc07964980c43b53c46a3d2ffee